Library/Formula/agda.rb

```diff
@@ -51,38 +56,81 @@ def install
     cabal_clean_lib
 
     if build.with? "stdlib"
-      resource("stdlib").stage prefix/"agda-stdlib"
+      resource("stdlib").stage prefix/"lib"
```

Comment:
Agda is a research language, with new releases introducing version changes. I’m not aware of any other formulae depending on Agda, and I don’t think there’s many user installations of Homebrewed Agda. Considering that prior to this patch, the Emacs mode didn’t work, the MAlonzo-based tests didn’t pass, and the formula wasn’t updated in the month following the [release of 2.4.2.4](http://wiki.portal.chalmers.se/agda/pmwiki.php).